src/concourse/pipelines/infrastructure/ovs/docker_packer_pulumi_pipeline.py

```diff
@@ -0,0 +1,194 @@
+#  noqa: WPS232
+import sys
+import textwrap
+
+from concourse.lib.constants import (
+    PULUMI_CODE_PATH,
+    PULUMI_WATCHED_PATHS,
+    REGISTRY_IMAGE,
+)
+from concourse.lib.jobs.infrastructure import packer_jobs, pulumi_jobs_chain
+from concourse.lib.models.fragment import PipelineFragment
+from concourse.lib.models.pipeline import (  # noqa: WPS235
+    AnonymousResource,
+    Command,
+    GetStep,
+    Identifier,
+    Input,
+    Job,
+    Output,
+    Pipeline,
+    Platform,
+    PutStep,
+    RegistryImage,
+    TaskConfig,
+    TaskStep,
+)
+from concourse.lib.resources import git_repo, registry_image
+
+
+def build_ovs_pipeline() -> Pipeline:
+    ovs_rc_branch = "release-canidate"
+    ovs_r_branch = "release"
+
+    ovs_rc_repo = git_repo(
+        Identifier("odl-video-service-rc"),
+        uri="https://github.com/mitodl/odl-video-service",
+        branch=ovs_rc_branch,
+    )
+
+    # This is only used to trigger deployment to production
+    # no artifacts from this resource should actaully be utilized
+    ovs_r_repo = git_repo(
+        Identifier("odl-video-service-release"),
+        uri="https://github.com/mitodl/odl-video-service",
+        branch=ovs_r_branch,
+    )
+
+    ovs_registry_image = registry_image(  # noqa: S106
+        name=Identifier("ovs-image"),
+        image_repository="mitodl/ovs-app",
+        username="((dockerhub.username))",
+        password="((dockerhub.password))",
+    )
+
+    ovs_packer_code = git_repo(
+        name=Identifier("ol-infrastructure-packer-build"),
+        uri="https://github.com/mitodl/ol-infrastructure",
+        branch="main",
+        paths=[
+            "src/bilder/images/odl_video_service",
+        ],
+    )
+
+    ovs_pulumi_code = git_repo(
+        name=Identifier("ol-infrastructure-pulumi-build"),
+        uri="https://github.com/mitodl/ol-infrastructure",
+        branch="main",
+        paths=PULUMI_WATCHED_PATHS
+        + [
+            PULUMI_CODE_PATH.joinpath("applications/odl_video_service/"),
+        ],
+    )
+
+    dcind_resource = AnonymousResource(
+        type=REGISTRY_IMAGE,
+        source=RegistryImage(repository="mitodl/concourse-dcind", tag="latest"),
+    )
+
+    # TODO MD 20230110
+    # May be able to convert this to something that uses 'concourse.lib.tasks.container_build_task'
```

Comment:
Yeah, the idea that I had when fixing up the Dockerfile was to avoid having to run this custom bash script and instead just do a simple Docker build, specifying the `production` step as the target.

Reply:
I'm going to leave it as a TODO for now because I don't think I have the patience at the moment for too much more debugging on this.